.env.example

```diff
@@ -1,3 +1,6 @@
 REACT_APP_API_DOMAIN=http://localhost:8000
 REACT_APP_ENV='dev'
 REACT_APP_DEVELOP_BUILD_VERSION=
+REACT_APP_LAUNCHDARKLY_CLIENT_ID=65df9142fbf14e10954ac55d
```

Comment:
Should this be added to the `README`? (Omitting the one below, since it's a temporary workaround)

Reply:
Yes, I didn't have time to write up documentation when I posted this draft but I'll be sure to include this new env

Reply:
Will leave LD documentation on Confluence, no need to replicate here. Just added the env to README